TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/LinearSlideSubsystem.java

```diff
@@ -55,17 +55,13 @@ public LinearSlideSubsystem(OpMode opMode) {
 
     public void runPIDPosition() {
 
-        /*if(motor.getCurrentPosition() > targetPosition - 5 && motor.getCurrentPosition() < targetPosition + 5) {
-            motor.setPower(0);
-            return;
-        }*/
         double outputPower = controller.calculate(motor.getCurrentPosition(), targetPosition);
         telemetry.addData("Power to motors:", outputPower);
 
 
         if(Math.abs(outputPower) > 0.6 ) {
             if (elapsedTime.seconds() > timeOut) {
-                motor.setPower(Constants.IntakeConstants.PID_SAFE_POWER);
+                motor.setPower(0);
                 telemetry.addLine("slide timeout triggered"); }
             else { motor.setPower(outputPower); }
```

Comment:
Why do I bother thoroughly reviewing your prs when you don't even read my comments!? I distinctly remember saying you really don't need those curly braces in line 66 in one of your earlier prs.

Reply:
Really sorry for not reading that part, and we don't need that {} as:
else motor.setPower(outputPower);